modules/api/src/main/java/io/fluo/api/config/FluoConfiguration.java

```diff
@@ -227,8 +228,8 @@ private void verifyApplicationName(String name) {
       if (c == 0) {
         reason = "null character not allowed @" + i;
         break;
-      } else if (c == '/' || c == '.') {
-        reason = "invalid character @" + i;
+      } else if (c == '/' || c == '.' || c == ':') {
```

Comment:
I looked around on the web to see if there are any other chars that hdfs does not allow.  I could not find anything definitive, but colon does seem to be one to avoid.

https://issues.apache.org/jira/browse/HADOOP-7945